bin/kubectl-crossplane-trace

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
What is the benefit of having a `bash` wrapper around another binary?

Reply:
As you know, kubectl plugins require an exact name matching the command/subcommand.
This way, we can use a single binary produced from the go code for different subcommands (currently only for trace but later we can consider adding more).